test/Microsoft.Health.Fhir.R4.Tests.E2E/Microsoft.Health.Fhir.R4.Tests.E2E.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <IsPackable>true</IsPackable>
     <RootNamespace>Microsoft.Health.Fhir.Tests.E2E</RootNamespace>
+    <IncludeContentInPack>false</IncludeContentInPack>
```

Comment:
@brandonpollett Thanks for the suggesting this to fix testHost.dll [issue](https://github.com/microsoft/fhir-server/pull/1208#discussion_r489032559) with the tests. I have tested Paas locally referring to the local packages after adding this tag and all tests continue to pass.